ui/src/main/java/com/kumuluz/ee/openapi/mp/ui/filters/CustomServer.java

```diff
@@ -0,0 +1,71 @@
+package com.kumuluz.ee.openapi.mp.ui.filters;
+
+import org.eclipse.microprofile.openapi.models.servers.Server;
+import org.eclipse.microprofile.openapi.models.servers.ServerVariable;
+import org.eclipse.microprofile.openapi.models.servers.ServerVariables;
+
+import java.util.Collections;
+import java.util.Map;
+
+/**
+ * Server implementation used to insert generated server into Swagger UI server list.
+ *
+ * @author Michal Vrsansky
+ */
+public class CustomServer implements Server {
+
+    private final String url;
+
+    public CustomServer(String url) {
+        this.url = url;
+    }
+
+    @Override
+    public String getUrl() {
+        return url;
+    }
+
+    @Override
+    public void setUrl(String url) {
+    }
+
+    @Override
+    public String getDescription() {
+        return "(added by auto-configuration)";
```

Comment:
If the server was listed before no notification "(added by auto-configuration)" should be added (if auto-configuration only changed the order of server entry).

Reply:
done